### PR TITLE
Improvement: new tags

### LINKED ISSUE #323: Use new tag
## Description

Use tag `language` for all plugin selection tag (CreateDiagramDrawer).
Use tag `category` for diagrams table catogory (DiagramsSearchCard).

### PR DESCRIPTION
**Pull request by Rahzbob**
Resolve #323

### CODE REVIEW

File: tests/unit/components/drawer/CreateDiagramDrawer.spec.js [hunk 1]
```diff
@@ -91,7 +102,7 @@ describe('Test component: CreateDiagramDrawer', () => {
 
     it('should return only githubator template', () => {
       wrapper.vm.searchTemplateText = null;
-      wrapper.vm.selectedPlugin = 'githubator-plugin';
+      wrapper.vm.selectedLanguage = 'Github';
```
Comment (k-y0u) [inline tests/unit/components/drawer/CreateDiagramDrawer.spec.js:105]: jump line after

File: src/composables/Project.js [hunk 2]
```diff
@@ -810,7 +810,7 @@ export async function getAllModels(projectId) {
         id: `diagram_${index}`,
         plugin: plugin.data.name,
         path,
-        tags: plugin.configuration.tags,
+        tags: plugin.configuration.tags.filter(({ type }) => type === 'category'),
```
Comment (k-y0u) [inline src/composables/Project.js:813]: why ?

File: src/components/card/DiagramsCard.vue [hunk 3]
```diff
@@ -120,7 +120,7 @@ const data = reactive({
 });
 const searchDiagramText = ref('');
 const selectedTags = ref([]);
-const allTags = ref(getAllTags());
+const allTags = ref(getAllTagsByType('category'));
```
Comment (k-y0u) [inline src/components/card/DiagramsCard.vue:123]: `allTags` -> `categoryTags`

File: changelog.md [hunk 4]
```diff
@@ -34,6 +34,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Rework and clean all layouts and pages.
 * Have only one configuration file by project to save all diagrams positions.
 * Save diagram template position.
+* Models page:
+  * Update tags in DiagramTable & CreateDiagramDrawer drawer.
```
Comment (k-y0u) [inline changelog.md:38]: rm `drawer`

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: src/components/drawer/CreateDiagramDrawer.vue [hunk 5]
```diff
@@ -57,13 +57,13 @@
...
-            v-model="selectedPlugin"
+            v-model="selectedLanguage"
             outlined
             dense
             clearable
             class="plugin-select"
             :label="$t('page.models.drawer.select')"
-            :options="plugins"
+            :options="languages"
             data-cy="plugin-select"
```
Comment (justinehell) [inline src/components/drawer/CreateDiagramDrawer.vue:67]: language-select

File: changelog.md (hunk 4 above)
Comment (Zorin95670) [reply changelog.md:38]: Reword in 
```
* Models page:
  * Rework on tag display, use tag category for all diagrams in table.
  * Rework on tag display, use tag language for plugin selection on create diagram from template.
```

**Review by justinehell — CHANGES_REQUESTED** [review]

File: src/components/drawer/CreateDiagramDrawer.vue (hunk 5 above)
Comment (justinehell) [reply src/components/drawer/CreateDiagramDrawer.vue:67]: also update class line 64

Comment (k-y0u) [reply src/components/drawer/CreateDiagramDrawer.vue:67]: same for `class` attribute

File: src/composables/Project.js (hunk 2 above)
Comment (Zorin95670) [reply src/composables/Project.js:813]: remove it

Comment (Rahzbob) [reply src/composables/Project.js:813]: Because otherwise it will also get the `Language` tags, which is not what we want in the `DiagramTable`.

**Review by k-y0u — DISMISSED** [review]

File: src/components/drawer/CreateDiagramDrawer.vue (hunk 5 above)
Comment (Rahzbob) [reply src/components/drawer/CreateDiagramDrawer.vue]: Done.

**Review by Zorin95670 — APPROVED** [review]

**Review by justinehell — APPROVED** [review]

**Pull request merged**